### ISSUE TITLE
lvm2 requires 2.03.16 upgrade to unblock csi-driver-lvm package usage

### ISSUE THREAD

**Issue by liulanze** [microsoft/CBL-Mariner#3979]
The csi-driver-lvm package requires lvm2 as one of the dependencies, while the current version 2.03.15 blocks the package to work properly, in this case to request your help on upgrading it.

Details here: when csi-driver-lvm creates logical volume, the device size of libudev will always be checked, and shows error bothering pv_min_size check failed. We want to clear this for the package to create logical volume properly.

Here is the link for lvm2 upgrading by upstream -> [filter-usable: remove udev dev size check · lvmteam/lvm2@deaf43d (github.com)](https://github.com/lvmteam/lvm2/commit/deaf43d6f0341d8d15e8ff5ffd25d7e0a7d2092a)

Please let us know if you have any questions and we would like to collaborate to work on this issue!

**Comment by oliviacrain** [comment]
@liulanze I'm a bit confused here- the linked commit is listed as present in the `v2_03_15` tag, which is the version we currently ship for Mariner 2.0.

We can work with you to upgrade the package or pull in necessary patches, but it would be nice to have some clarity as to which fix we actually need to pull in.

**Comment by liulanze** [comment]
Hi @oliviacrain Thanks for the message. I am currently under investigation and will get back the result here soon. Request to pending this issue temporarily. Thanks.

**Comment by liulanze** [comment]
Close this one since found out the lvm2 version is good for now.
